test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,58 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+require('dotenv').config();
+
+const instance = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+const urlBase = 'https://api.github.com/user';
+const urlIssue = 'https://api.github.com/repos';
+let selectedRepo;
+let issue;
+
+describe('Consume Post&Patch', () => {
+  it('Logged User', async () => {
+    const loggedUser = await instance.get(`${urlBase}`);
+
+    expect(loggedUser.status).to.equal(StatusCodes.OK);
+    expect(loggedUser.data.public_repos).to.be.greaterThanOrEqual(1);
+  });
+
+  it('Get a Repository', async () => {
+    const repos = await instance.get(`${urlBase}/repos`);
```

Comment:
Try to use hypermedia properties, with something like:
```suggestion
    const repos = await instance.get(loggedUser.data.repos_url);
```